cmd/root.go

```diff
@@ -4,7 +4,12 @@ import (
 	"go.uber.org/zap"
 	"github.com/spf13/cobra"
 
-	"vogelkop/internal"
+	"github.com/metal-toolbox/vogelkop/internal"
+
+	"strconv"
+	"os/exec"
+	"strings"
+	"path/filepath"
```

Comment:
this looks pretty weird, most go files are just 2 import blocks. The stdlib stuff at the top and everything else on a second block. `gofumpt` doesn't quite fix this yet I think though :(.